source/codegen/metadata_validation.py

```diff
@@ -205,7 +207,10 @@ def validate_metadata(metadata: dict):
         attribute_enums = _get_attribute_enums(metadata)
         used_enums = function_enums.union(attribute_enums)
         for enum_name in metadata["enums"]:
-            _validate_enum(enum_name, used_enums, metadata)
+            if not _rule_is_suppressed(
+                metadata, RULES.ENUMS_SHOULD_BE_USED, ["enums", enum_name]
+            ):
```

Comment:
```suggestion
            if not _rule_is_suppressed(metadata, RULES.ENUMS_SHOULD_BE_USED, ["enums", enum_name]):
```
We have Black do a formatting check on our .py files. I believe it's failing the build on this line. Ran black locally on your changes and it seems to prefer one line for this if statement.

Reply:
Thanks for the suggestion Ryan. I have configured black and formatted.